app/src/main/java/com/google/samples/apps/topeka/widget/quiz/FourQuarterQuizView.java

```diff
@@ -91,10 +91,15 @@ public void onLayoutChange(View v, int left, int top,
     }
 
     private void setUpUserInput() {
-        mAnswerView.performItemClick(mAnswerView.getChildAt(mAnswered), mAnswered,
-                mAnswerView.getAdapter().getItemId(mAnswered));
-        mAnswerView.getChildAt(mAnswered).setSelected(true);
-        mAnswerView.setSelection(mAnswered);
+        mAnswerView.post(new Runnable() {
+            @Override
+            public void run() {
+                mAnswerView.requestFocusFromTouch();
```

Comment:
The same method, but changed name of view. Extracting to method with parameters `View mAnswerView`and `int mAnswered` could result in reusing this method in classes below.